src/linearmixedmodel.jl

```diff
@@ -775,21 +775,22 @@ function reevaluateAend!(m::LinearMixedModel)
 end
 
 """
-    refit!(m::LinearMixedModel[, y::Vector]; REML=m.optsum.REML)
+    refit!(m::LinearMixedModel[, y::Vector]; REML=m.optsum.REML, progress::Bool=false)
```

Comment:
```suggestion
    refit!(m::LinearMixedModel[, y::Vector]; REML=m.optsum.REML, progress::Bool=true)
```

for consistency with `fit`